### DISCUSSION TITLE
apollo1.6.1源码打包后读取不到外部配置

### DISCUSSION THREAD

**Discussion by gy448** [apolloconfig/apollo#3770] [Q&A] [answered]
apollo1.6.1源码打包后，启动时报如下错误：
***************************
APPLICATION FAILED TO START
***************************
Description:
Failed to configure a DataSource: 'url' attribute is not specified and no embedded datasource could be configured.
Reason: Failed to determine a suitable driver class
Action:
Consider the following:
	If you want an embedded database (H2, HSQL or Derby), please put it on the classpath.
	If you have database settings to be loaded from a particular profile you may need to activate it (no profiles are currently active).
将数据源配到yml文件中就可以读取到，如何打包apollo源码使之能读取到外部配置的数据源？

**Comment by nobodyiam** [comment, marked as answer]
you need to use the build.sh to package apollo, it will activate `github` profile
